Makefile.am

```diff
@@ -25,13 +25,15 @@ libyaksa_la_LDFLAGS = -version-info @libyaksa_so_version@
 
 include $(top_srcdir)/src/Makefile.mk
 
-testlists=
+test_deps = libyaksa.la test/dtpools/libdtpools.la
+
+testlists =
```

Comment:
Unrelated white space cleanup.  Should be in a separate patch.

Reply:
fixed.